content/docs/latest/installing/cloud/openstack.md

```diff
@@ -51,10 +51,12 @@ $ bunzip2 flatcar_production_openstack_image.img.bz2
 Once the download completes, add the Flatcar Container Linux image into Glance:
 
 ```shell
-$ glance image-create --name Container-Linux \
+$ openstack image create \
   --container-format bare \
   --disk-format qcow2 \
-  --file flatcar_production_openstack_image.img
+  --property hw_qemu_guest_agent=yes \
```

Comment:
I was a bit opaque so why I would add this:

- easier snapshots
- easier shutdown/reboot (of VMs)

LBNL, flatcar has it build in. :)

Reply:
It does work, we are using it.